### ISSUE TITLE
Please add support for custom Slack message bodies

### ISSUE THREAD

**Issue by broderick99** [target/goalert#3336]
**What problem would you like to solve? Please describe:**
We mainly rely on automated Slack alerts to remind people that they are on call. We want to include more information in the message to make the on-call experience go smoother.

**Describe the solution you'd like:**
We want the flexibility of being able to customize the alert body of a GoAlert. The use case is for our on-call message. We want to be able to add reminders and instructions for the person that is on-call.

**Describe alternatives you've considered:**
There are no alternatives

**Additional context:**
This would be a really useful add-on! Please let us know if we can provide any more context.

**Comment by mastercactapus** [comment]
Thanks for the suggestion -- I assume this is for the On-Call Notifications on a schedule?

Would it suffice to have an "additional details" or something (i.e., something to tack on to the end of the message) or were you thinking something different?

**Comment by broderick99** [comment]
@mastercactapus Yes, this is for on-call notifications. I think that an additional details field that would just tack on extra content to the message that goes out would work great. Hoping for whatever is easy :D
